src/updates/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: layouts/default.njk
+title: Updates
+---
+
+Open collective community updates are posted <a href="https://opencollective.com/zinc-community/updates" target="_blank">here</a>.
```

Comment:
Maybe `News and Updates are posted Quarterly-ish on our [Community Open Collective's Updates page](...).`?

Using "here" as link-text is not great for accessibility. Also, I'm curious about your thoughts on [this article on `target="_blank"`](https://css-tricks.com/use-target_blank/)?

Reply:
text fixed in https://github.com/zinc-collective/www.zinc.coop/pull/107/commits/24ddfbc056c6b7d360774e8991c9e29e194ed53b

Good article. Makes a really good point! I agree with allowing users the option of both opening a link in the current window and opening a link in a tab via holding Cmd key on mac/equivalent key in Windows.

thus I removed finance links opening in a new tab in https://github.com/zinc-collective/www.zinc.coop/pull/107/commits/939a61a2f17f97259e616738906d4599ec400e77